extensions/ql-vscode/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## 1.0.6
+## 1.0.6 - 28 February 2020
 
 - Add command to restart query server.
+- Improve cli upgrade versioning.
```

Comment:
Maybe something like:

```suggestion
- Enables support for future versions of the CodeQL CLI.
```

to explain what's changed?

Reply:
Future major/minor versions, perhaps.

Reply:
I'm confused about this comment; it specifically allows minor-version upgrades, but still mins the major version number.

Reply:
Ah that should have said just 'minor'.